### PR TITLE
About item page

### PR DESCRIPTION
**Pull request by kinqbert**

### CODE REVIEW

**Review by metrakonon — APPROVED** [review]
Great work! :fire:

File: src/pages/AboutItemPage/AboutItemPage.scss [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+
+  margin-top: 56px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  grid-column: 1 / -1;
+
+  background-color: #e2e6e9;
```
Comment (ilyasolo881) [inline src/pages/AboutItemPage/AboutItemPage.scss:361]: Will be good use variable

File: src/pages/AboutItemPage/AboutItemPage.scss [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+  padding: 2px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: $white;
+
+  border-radius: 50%;
+  border: 1px solid #e2e6e9;
```
Comment (ilyasolo881) [inline src/pages/AboutItemPage/AboutItemPage.scss:410]: And here

File: src/pages/AboutItemPage/AboutItemPage.scss [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+
+  cursor: pointer;
+
+  transition: border 0.3s;
+
+  &:hover {
+    border: 1px solid #b4bdc3;
```
Comment (ilyasolo881) [inline src/pages/AboutItemPage/AboutItemPage.scss:417]: And here :)

File: src/pages/AboutItemPage/AboutItemPage.scss [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+  }
+
+  &--selected {
+    border: 1px solid #313237;
```
Comment (ilyasolo881) [inline src/pages/AboutItemPage/AboutItemPage.scss:421]: U know :)

**Review by ilyasolo881 — APPROVED** [review]
Nice work!

**Pull request merged**